Log helpful message if `alpinejs` is not found

`alpinejs` is a peer dependency.

If it's missing, users will see the following error:

```
Error: Cannot find module 'alpinejs'
```

Followed by a call stack.

Instead of a random `module` message, let's log a friendly message hinting that the user should `npm i --save-dev alpinejs` (or `yarn add --dev alpinejs`).

--- a/src/main.js
+++ b/src/main.js
@@ -8,7 +8,14 @@ const {config, setGlobal, setMutationObserver} = require('./config');
 // Needs to happen before loading Alpine
 config();
 
-const Alpine = require('alpinejs');
+let Alpine;
+try {
+  Alpine = require('alpinejs');
+} catch (_) {
+  throw new Error(
+    "Alpine.js npm module ('alpinejs') not found - try installing it with `npm install --save-dev alpinejs`"
+  );
+}
 
 const getComponents = (markup) => {
   const {document} = new JSDOM(markup).window;

